lib/module/comments/comments_presenter.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/foundation.dart';
+
+import 'package:flutter_news/injection/dependency_injection.dart';
+import 'package:flutter_news/model/hn_item.dart';
+
+abstract class CommentsViewContract {
```

Comment:
I agree it currently looks a bit silly :) I'm trying to adhere to that MVP pattern here. Abstract class is useful in IDEA since it would give a warning that you need to implement abstract class methods and even prepare empty method overrides for you. I think it also might be easier to implement should you decide to add more view contract methods later.